server/service/src/errors.rs

```diff
@@ -0,0 +1,17 @@
+use repository::RepositoryError;
```

Comment:
there are some other `errors` files so I thought to do the same. Just needed somewhere common to both modules which are using this error